src/views/Player.vue

```diff
@@ -86,9 +86,19 @@ export default defineComponent({
       type: String,
       required: true,
     },
+    userId: {
+      default: null,
+      type: String,
+    },
+    apiKey: {
+      default: null,
+      type: String,
+    },
```

Comment:
same comment as above - why do we need them as prop - can directly read the query param as we've been reading userId before